zulipterminal/ui.py

```diff
@@ -79,6 +79,12 @@ def footer_view(self) -> Any:
         text_header = self.get_random_help()
         return urwid.AttrWrap(urwid.Text(text_header), 'footer')
 
+    def show_updated_right_column(self, model: Any) -> Any:
+        self.users = model.users
+        self.right_panel = self.right_column_view()
+        self.main_window()
+        self.controller.update_screen()
```

Comment:
We discussed the issues you were having here on chat.zulip.org; I suspect the easier solution to integrate with the existing interface is probably to avoid adding this method and instead use `RightColumnView.update_user_list`, possibly adjusted slightly.

Reply:
Yes, this is the proper way to update the list. We don't want want re-initialising the view every minute might have other problems like what happens if the user is searching for users and we re-initialise the view?

Reply:
Thanks for the tip @neiljp and @amanagr! I have tweaked `update_user_list` to display the updated user list. Let me know how the PR looks.